tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/demo/tf-interactive-inference-age-demo.html

```diff
@@ -178,60 +188,100 @@
                            'Federal-gov', 'Local-gov', '?']}];
           });
           this.$.dash.addEventListener('infer-mutants', async e => {
-            const examples = [];
-            const featureMapping = {
-              'capital-gain': 0,
-              'capital-loss': 1,
-              'education': 2,
-              'education-num': 3,
-              'hours-per-week': 4,
-              'marital-status': 5,
-              'native-country': 6,
-              'occupation': 7,
-              'race': 8,
-              'relationship': 9,
-              'sex': 10,
-              'workclass': 11
-            };
-            const xMin = +e.detail.x_min;
-            const xMax = +e.detail.x_max;
-            const isNum = !this.$.dash.partialDepPlotEligibleFeatures[
-              featureMapping[e.detail.feature_name]].samples;
-            const numTrials = isNum ? 10 :
-              this.$.dash.partialDepPlotEligibleFeatures[featureMapping[
-                e.detail.feature_name]].samples.length;
-            for (let i = 0; i < numTrials; i++) {
-              const ex =
-              JSON.parse(JSON.stringify(this.data[e.detail.example_index]));
-              if (isNum) {
-                ex.features.feature[e.detail.feature_name].floatList.value[0] =
-                  xMin + i * (1 / (numTrials - 1) *(xMax - xMin));
-              } else {
-                ex.features.feature[e.detail.feature_name].bytesList.value[0] =
-                  btoa(this.$.dash.partialDepPlotEligibleFeatures[
-                    featureMapping[e.detail.feature_name]].samples[i]);
+            const method = async () => {
```

Comment:
I don't think this method has to be `async` since it seems to be just a callback to `setTimeout`. Just curious, what is the intention?